app/components/room-players-table.tsx

```
@@ -0,0 +1,42 @@
+  Table,
+  TableBody,
+  TableCaption,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from "@/components/ui/table";
+
+type Props = {};
+
+export default function RoomPlayersTable({}: Props) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)